lib/class-vip-request-block.php

```diff
@@ -26,6 +26,8 @@ class VIP_Request_Block {
 	 * @return bool|void
 	 */
 	public static function ip( string $value ) {
+		$value = strtolower( $value );
```

Comment:
If PHP is compiled without IPv6, we fall back to case-insensitive string comparisons.